program/src/instruction.rs

```diff
@@ -419,8 +425,6 @@ accounts_struct! {
             is_signer: false,
             is_writable: false,
         },
-        const sysvar_clock = sysvar::clock::id(),
-        const sysvar_rent = sysvar::rent::id(),
```

Comment:
We can’t modify existing instructions; that is a breaking change for all existing things that call this instruction due to https://github.com/ChorusOne/solido/blob/b4ba3f7919b9f07b4dad6be7fca1c290467a5328/program/src/accounts.rs#L237-L240

The proper way to deal with this is to append a new instruction, `UpdateExchangeRateV2`, that is the same as v1, but with these two input accounts omitted. In this case, because the new implementation needs only a subset of the accounts, we can transparently handle this in the instruction dispatcher, a call to `UpdateExchangeRate` v1 would construct an `UpdateExchangeRateV2AccountsInfo` struct and then call the implementation for v1.